addons/website_sale/controllers/main.py

```diff
@@ -1166,10 +1173,13 @@ def products_autocomplete(self, term, options={}, **kwargs):
         order = self._get_search_order(options)
         max_nb_chars = options.get('max_nb_chars', 999)
 
-        category = options.get('category')
         attrib_values = options.get('attrib_values')
 
-        domain = self._get_search_domain(term, category, attrib_values, display_description)
+        domain = self._get_search_domain(term, attrib_values, display_description)
+        category = options.get('category')
```

Comment:
missing update docstring

Reply:
in fact category in not even a possibility here there are no rpc calling it with the argument. Will remove it.